packages/nextjs/scaffold.config.ts

```diff
@@ -4,11 +4,16 @@ type ScaffoldConfig = {
   targetNetwork: chains.Chain;
   pollingInterval: number;
   alchemyApiKey: string;
+  burnerWallet: {
+    enabled: boolean;
+    onlyLocal: boolean;
+  };
+  walletAutoConnect: boolean;
 };
 
 const scaffoldConfig = {
   // The network where your DApp lives in
-  targetNetwork: chains.hardhat,
+  targetNetwork: chains.goerli,
```

Comment:
Don't forget to put it back to hardhat when PR is finalized :)

Reply:
Tysm 🙌 Fixed at [8c7fa2c](https://github.com/scaffold-eth/se-2/pull/257/commits/8c7fa2c00c60ff1b266c5b228d4b6b38580b5a68)